ui/src/publication/publication-utils.ts

```diff
@@ -71,19 +71,19 @@ export const createPublicationCandidateReference = (
 ): PublicationCandidateReference => {
     switch (type) {
         case DraftChangeType.TRACK_NUMBER:
-            return { id: brand(id), type };
+            return {id: brand(id), type};
```

Comment:
Onko prettier päivittynyt tms.? En muista että se olisi aiemmin tyylitellyt näitä näin 🤔

Reply:
Mitenköhän tämä on edes mahdollista 😄, kokeilin tallentaa tiedoston uudelleen niin formatoi nuo jälleen uusiksi. Prettier ajautuu kuitenkin tallennuksen yhteydessä enkä mä tuota filua ole muualla kuin idean kautta editeerannut. Fiksaan takas ->